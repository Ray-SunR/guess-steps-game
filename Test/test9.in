gam 	
gama
		

game 2 1 test9A1.txt test9B1.txt
quia
quib
quic
quit
