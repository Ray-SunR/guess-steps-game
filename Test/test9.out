Total is 2, max deduction is 1
Player A's move
Total is 1
Player B's move
Total is 0
Player B wins
Score is
A 0
B 1
